code/modules/events/sudden_paranoia.dm

```diff
@@ -0,0 +1,24 @@
+/datum/round_event_control/renegade
```

Comment:
I don't really want random events that trigger light antagonists, I'm more amicable to ones for larger antagonists but on its own I don't want this

Reply:
> I don't really want random events that trigger light antagonists, I'm more amicable to ones for larger antagonists but on its own I don't want this

We already have one for Obssesed thats why I copied it, I thought as this is too light brain trauma antag it can be like this too. So I should delete the Obsessed one too?